### PR TITLE
GraphQL-431: Test coverage: Using the "use_for_shipping" option with multishipping is not possible

### PR DESCRIPTION
**Pull request by AleksLi**
Covered SetBillingAddressOnCartTest with multiple shipping addresses in a quote. Added Fixture for that.

### Fixed Issues (if relevant)

1. magento/graphql-ce#431: Test coverage: Using the "use_for_shipping" option with multishipping is not possible

### Manual testing scenarios (*)

1. Run the tests of SetBillingAddressOnCartTest for all the customer types Guest|Saved Customer.

### CODE REVIEW

File: dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_addresses_saved.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
```
Comment (naydav) [inline dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_addresses_saved.php]: We already the same fixture
Please, check the same class, it should be used in another testcases

BTW if we create fixture we MUST create rollback
Thanks

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php [hunk 2]
```diff
@@ -403,6 +403,51 @@ public function testSetBillingAddressIfCustomerIsNotOwnerOfAddress()
         $this->graphQlQuery($query, [], '', $this->getHeaderMap('customer2@search.example.com'));
```
Comment (naydav) [inline dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php]: Need to add the same case for Guest
https://github.com/magento/graphql-ce/blob/3469f4b251cdc4874048dfff0374f818cb757964/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest/SetBillingAddressOnCartTest.php

**Review by naydav — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_addresses_saved.php (hunk 1 above)
Comment (AleksLi) [reply dev/tests/integration/testsuite/Magento/Checkout/_files/quote_with_multiple_addresses_saved.php]: @naydav I haven't found it. Can you take a look and give me the name. I did look at the whole project.

File: dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php (hunk 2 above)
Comment (AleksLi) [reply dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer/SetBillingAddressOnCartTest.php]: @naydav Yes, I will do that

File: dev/tests/integration/testsuite/Magento/Checkout/_files/guest_quote_with_multiple_addresses_saved.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
+<?php
```
Comment (naydav) [inline dev/tests/integration/testsuite/Magento/Checkout/_files/guest_quote_with_multiple_addresses_saved.php]: Please, reuse current fixtures files
You can find examples for all of the cases in

https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Customer

https://github.com/magento/graphql-ce/tree/2.3-develop/dev/tests/api-functional/testsuite/Magento/GraphQl/Quote/Guest
(also in the same testcase class)

**Review by naydav — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/Checkout/_files/guest_quote_with_multiple_addresses_saved.php (hunk 3 above)
Comment (AleksLi) [reply dev/tests/integration/testsuite/Magento/Checkout/_files/guest_quote_with_multiple_addresses_saved.php]: @naydav  ok. I'll do that. Thanks.

**Comment by naydav** [comment]
@AleksLi 
Implementation is overhead. You should not use create or modify quote or address model

What need to do:
1) Use the `quote with addresses` fixture as a precondition (could be found in the same file)
2) During test try to assign billing address with `use_for_shipping=true` flag
3) Check error message

Thanks

**Comment by AleksLi** [comment]
@naydav please take a look on this one again. I know that some of the checks are FAILED but I don't know why, actually. I've seen the reason and it looks weird as for me.

**Comment by naydav** [comment]
Was covered in the scope of
https://github.com/magento/graphql-ce/pull/321
https://github.com/magento/graphql-ce/pull/330

**Pull request closed without merging**